docs/guide/index.asciidoc

```diff
@@ -56,6 +56,48 @@ To visualize the data after it's sent to Elasticsearch,
 you can use the the dedicated APM UI bundled in X-Pack,
 or the pre-built open source Kibana dashboards that can be loaded directly in the Kibana UI.
 
+[[apm-release-notes]]
+=== Release notes
+
+[float]
+==== APM version 6.4
+
+[float]
+===== Breaking changes
+
+We previously split APM data into separate indices (transaction, span, error, etc.).
+In 6.4 APM Kibana UI starts to leverage those separate indices for queries.
+In case you only update Kibana but run an older version of APM Server you will not be able to see any APM data by default.
+To fix this, use the Kibana APM configuration options to specify the location of the APM index.
+
+When upgrading to Kibana 6.4, you might need to refresh your APM index pattern for certain APM UI features to work.
+
+[float]
+===== New features
```

Comment:
I think it would make sense to link to the according changelog sections or if not available to the overview for Server, UI and Agents.

Reply:
agents are not adding their changelogs to the Elastic docs, and I'd prefer not to link to GH because we can't detect broken links. 
(also personally I don't think it looks very nice to follow a link from elastic docs to a GH readme, even thou i don't know if others are doing that)